README.md

```diff
@@ -1,10 +1,32 @@
+
+ENGLISH >>>>>>
+
+# Final Project Henry - Co-debug
+
+The site is designed to provide a space for questions and answers among programming students. It is planned to be dynamic and intuitive so that the experience is as spontaneous as possible. Unlike other pre-existing solutions.
+Among the functions to be developed, we can highlight the evaluation of answers to questions with more time without solving. We rate those who help and highlight them in a ranking according to their participation. We also make interactions public so users can learn from others' questions.
```

Comment:
Cambiaría el inicio del párrafo a  "Among its functions, we can highlight the valuation of replies to questions with more time without being answered"